src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -144,6 +144,8 @@
             + "The value of '${fieldName}' field should be no longer than ${maxLength} characters.";
     public static final String INVALID_NAME_ERROR_MESSAGE =
             ERROR_INFO + " " + HINT_FOR_CORRECT_FORMAT_FOR_INVALID_NAME;
+    public static final String TEAM_NAME_IS_VALID_EMAIL_ERROR_MESSAGE =
+            "The field " + TEAM_NAME_FIELD_NAME + " is not acceptable to TEAMMATES as team name can't be a valid email";
```

Comment:
"As team name can't be a valid email" sounds quite weird. Suggest the following rephrase:

```txt
... is not acceptable to TEAMMATES as the suggested value for team name can be mis-interpreted as an email.
```

Also, `team name` is the value for the constant `TEAM_NAME_FIELD_NAME` so let's reuse it.